honeybee_grasshopper_energy/src/HB Apply Opaque Construction.py

```diff
@@ -0,0 +1,107 @@
+# Honeybee: A Plugin for Environmental Analysis (GPL)
+# This file is part of Honeybee.
+#
+# Copyright (c) 2019, Ladybug Tools.
+# You should have received a copy of the GNU General Public License
+# along with Honeybee; If not, see <http://www.gnu.org/licenses/>.
+# 
+# @license GPL-3.0+ <http://spdx.org/licenses/GPL-3.0+>
+
+"""
+Apply OpaqueConstruction to Honeybee Faces, Doors or Room walls.
+_
+This component supports the assigning of different constructions based on cardinal
+orientation, provided that a list of OpaqueConstructions are input to the _constr. 
+-
+
+    Args:
+        _hb_obj: Honeybee Faces, Doors or Rooms to which the input _constr should
+            be assigned. For the case of a Honeybee Room, the construction
+            will only be applied to the Room's outdoor walls. Note that, if you
+            need to assign a construction to all the roofs, floors, etc. of a
+            Room, the best practice is to create a ConstructionSet and assing that
+            to the Room.
+        _constr: A Honeybee OpaqueConstruction to be applied to the input _hb_obj.
```

Comment:
How does the user make the decision that if the construction should be applied to Doors and not the Walls if that is desired?